### ISSUE TITLE
Traefik unable to create logs folder in docker swarm

### ISSUE THREAD

**Issue by keshavpeswani** [containous/traefik#1217]
### What version of Traefik are you using (`traefik version`)?
1.1.2

### What is your environment & configuration (arguments, toml...)?
We are running traefik as docker service in swarm mode.

### What did you do?
 The following docker command is used to run traefik

```
sudo docker service create \
    --name traefik \
    --replicas=1 \
    --constraint=node.role==manager \
    --publish 80:80 --publish 8080:8080 \
    --mount type=bind,source=/var/run/docker.sock,target=/var/run/docker.sock \
    --network swarm-demo-overlay-net \
    traefik:v1.1.2 \
    --docker \
    --docker.swarmmode \
    --docker.domain=abc.com \
    --docker.watch \
    --logLevel=INFO \
    --traefikLogsFile='/app/logs/traefik.log' \
    --accessLogsFile='/app/logs/access.log' \
    --web

```
### What did you expect to see?
Traefik should create the log folder '/app/logs/'  if not present.

### What did you see instead?
Traefik throws following error at stdout and then redirects the logs to stdout

```shell
time="2017-03-02T04:13:55Z" level=error msg="Error opening fileopen \"/app/logs/access.log\": no such file or directory" 
time="2017-03-02T04:13:55Z" level=error msg="Error opening fileopen \"/app/logs/traefik.log\": no such file or directory" 
time="2017-03-02T04:13:55Z" level=info msg="Traefik version v1.1.2 built on 2016-12-15_10:21:15AM" 
time="2017-03-02T04:13:55Z" level=error msg="Error opening fileopen \"/app/logs/access.log\": no such file or directory" 

```

**Comment by DaBlitzStein** [comment]
Same here

**Comment by ldez** [comment]
Fixed by #1507
